staging/src/k8s.io/kubectl/pkg/cmd/set/set_subject.go

```diff
@@ -113,7 +113,7 @@ func NewCmdSubject(f cmdutil.Factory, streams genericclioptions.IOStreams) *cobr
 	cmd.Flags().StringVarP(&o.Selector, "selector", "l", o.Selector, "Selector (label query) to filter on, not including uninitialized ones, supports '=', '==', and '!='.(e.g. -l key1=value1,key2=value2)")
 	cmd.Flags().BoolVar(&o.Local, "local", o.Local, "If true, set subject will NOT contact api-server but run locally.")
 	cmdutil.AddDryRunFlag(cmd)
-	cmd.Flags().StringArrayVar(&o.Users, "user", o.Users, "Usernames to bind to the role")
+	cmd.Flags().StringArrayVar(&o.Users, "users", o.Users, "Usernames to bind to the role")
```

Comment:
it is a breaking change. probably best to keep the old flag for 1 (or more releases) and add a new flag "users" that would eventually replace it.

Reply:
Well pointed out @neolit123. Added the new flag and maintained the old one. Should we note somewhere that this would be removed in a future release?

Reply:
The kubectl maintainers should decide on the timeline.